docs/diagrams/AttendanceClassDiagram.puml

```diff
@@ -0,0 +1,32 @@
+@startuml
+!include style.puml
+skinparam arrowThickness 1.1
+skinparam arrowColor black
+skinparam classBackgroundColor ATTENDANCE_COLOR
+skinparam classFontColor black
+show member
+hide method
+
+Package Attendance <<Rectangle>>{
+
+Class AttendanceRecordList {
+recordList : List<AttendanceRecord>
+}
+
+Class AttendanceRecord {
+attendances : Map<UUID, Attendance>
+}
+
+Class Attendance {
+participationScore : int
+}
+
+Class Week {
+week : Index
+}
+
+AttendanceRecord "n" <-- AttendanceRecordList
+Attendance "*" <-- AttendanceRecord
```

Comment:
Perhaps this line isn't necessary since the association has been represented as a field already? I'm not too sure on how maps should be represented as well. We should probably check with Prof Damith for clarification.

Reply:
I think I will leave it like that for now, because I want to show `Attendance`. Will clarify with Prof Damith or Clarence.